src/transform/FooterReadMore.js

```diff
@@ -0,0 +1,337 @@
+  }
+
+  let description
+  if (readMorePage.terms) {
+    description = readMorePage.terms.description[0]
+  }
+  if ((description === undefined || description.length < 10) && readMorePage.extract) {
+    description = cleanExtract(readMorePage.extract)
+  }
+  if (description) {
+    const descriptionEl = document.createElement('div')
+    descriptionEl.id = index
+    descriptionEl.className = 'pagelib_footer_readmore_page_description'
+    descriptionEl.innerHTML = description
+    innerDivContainer.appendChild(descriptionEl)
+  }
+
+  const saveButton = document.createElement('div')
+  saveButton.id = `${_saveButtonIDPrefix}${encodeURI(readMorePage.title)}`
+  saveButton.className = 'pagelib_footer_readmore_page_save'
+  saveButton.addEventListener('click', event => {
+    event.stopPropagation()
+    event.preventDefault()
+    saveButtonClickHandler(readMorePage.title)
+  }, false)
+  innerDivContainer.appendChild(saveButton)
+
+  return document.createDocumentFragment().appendChild(outerAnchorContainer)
+}
+
+/**
+ * @type {ShownReadMorePagesHandler}
+ */
+const showReadMorePages = (pages, containerID, saveButtonClickHandler, titlesShownHandler,
+  document) => {
+  const shownTitles = []
+  const container = document.getElementById(containerID)
+  pages.forEach((page, index) => {
+    const title = page.title.replace(/ /g, '_')
+    shownTitles.push(title)
+    const pageModel = new ReadMorePage(title, page.thumbnail, page.terms, page.extract)
+    const pageFragment =
+      documentFragmentForReadMorePage(pageModel, index, saveButtonClickHandler, document)
+    container.appendChild(pageFragment)
+  })
+  titlesShownHandler(shownTitles)
+}
+
+/**
+ * Makes 'Read more' query parameters object for a title.
+ * @param {!string} title
+ * @param {!number} count
+ * @return {!Object}
+ */
+const queryParameters = (title, count) => ({
+  action: 'query',
+  continue: '',
+  exchars: 256,
+  exintro: 1,
+  exlimit: count,
+  explaintext: '',
+  format: 'json',
+  generator: 'search',
+  gsrinfo: '',
+  gsrlimit: count,
+  gsrnamespace: 0,
+  gsroffset: 0,
+  gsrprop: 'redirecttitle',
+  gsrsearch: `morelike:${title}`,
+  gsrwhat: 'text',
+  ns: 'ppprop',
+  pilimit: count,
+  piprop: 'thumbnail',
+  pithumbsize: 120,
+  prop: 'pageterms|pageimages|pageprops|revisions|extracts',
+  rrvlimit: 1,
+  rvprop: 'ids',
+  wbptterms: 'description',
+  formatversion: 2
+})
+
+/**
+ * Converts query parameter object to string.
+ * @param {!Object} parameters
+ * @return {!string}
+ */
+const stringFromQueryParameters = parameters => Object.keys(parameters)
+  .map(key => `${encodeURIComponent(key)}=${encodeURIComponent(parameters[key])}`)
+  .join('&')
+
+/**
+ * URL for retrieving 'Read more' pages for a given title.
+ * Leave 'baseURL' null if you don't need to deal with proxying.
+ * @param {!string} title
+ * @param {?string} baseURL
+ * @return {!sring}
+ */
+const readMoreQueryURL = (title, baseURL) => {
+  const readMoreItemFetchCount = 3
```

Comment:
I like that you've made a nice symbol for this value but also consider exposing it as a parameter.